Zotero/Scenes/Master/Settings/Storage/Views/StorageSettingsRow.swift

```diff
@@ -34,7 +34,7 @@ struct StorageSettingsRow: View {
     private func storageDataString(for data: DirectoryData?) -> String {
         guard let data = data, data.fileCount > 0 else { return "-" }
         let mbString = String(format: "%.2f", data.mbSize)
-        return (data.fileCount == 1 ? L10n.Settings.Storage.oneFile : L10n.Settings.Storage.multipleFiles(data.fileCount)) + " (\(mbString) MB)"
+        return (L10n.Settings.Storage.files(data.fileCount)) + " (\(mbString) MB)"
```

Comment:
Can we include the `(\(mbString) MB)` in that localized string too?

Reply:
Check https://github.com/zotero/zotero-ios/pull/735/commits/72345d1a83b3648e2b9fc0c3d37cee155306d87d